packages/core/src/api/toDecimal.ts

```diff
@@ -32,9 +41,15 @@ export function toDecimal<TAmount, TOutput>(calculator: Calculator<TAmount>) {
     assert(isDecimal, NON_DECIMAL_CURRENCY_MESSAGE);
 
     const units = toUnitsFn(dineroObject);
+    const isNegative = units.some((unit) => lessThanFn(signFn(unit), zero));
```

Comment:
Not sure you need `sign`. Comparing `unit` against `zero` should already work.

Reply:
Yes, this looks pretty funny in hindsight! I started with `sign`, then realized it returns a `TAmount` value, and then introduced the `lessThan` check to handle it. 😄